apps/chat/agent/tools.py

```diff
@@ -97,15 +97,15 @@ def action(
         minute: int,
         user_specified_custom_date: bool,
     ):
-        message = ScheduledMessage.objects.get(participant=self.experiment_session.participant, external_id=message_id)
-        if message.was_created_by_system and user_specified_custom_date:
```

Comment:
why revert this?

Reply:
I wanted the user to not be able to specify a new date, but went against that again. https://github.com/dimagi/open-chat-studio/pull/599/commits/0e6447c08f989c55d80630da664407a35209629f